lib/client/bgppeer.go

```diff
@@ -73,67 +76,62 @@ func (h *bgpPeers) Get(metadata api.BGPPeerMetadata) (*api.BGPPeer, error) {
 // that match the Metadata (wildcarding missing fields).
 func (h *bgpPeers) List(metadata api.BGPPeerMetadata) (*api.BGPPeerList, error) {
 	l := api.NewBGPPeerList()
-	err := h.c.list(metadata, h, l)
-	return l, err
+
+	// Global and host peers are listed separately.  Work out which we need
+	// to list.
+	listGlobal := (metadata.Scope == scope.Global || metadata.Scope == scope.Undefined) && metadata.Node == ""
```

Comment:
Shouldn't we return an error for `metadata.Scope == scope.Global && metadata.Node != ""`?